addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -32,6 +34,16 @@ <H3>Spider Add-On</H3>
 Other add-ons which use Traditional Spider functionality have also been re-worked to support the Spider add-on, including: Quick Start, Form Handler, GraphQL, OpenAPI, SOAP, and the Automation Framework. 
 More details are given below.
 
+<H3>Import/Export Add-On</H3>
```

Comment:
I think this one worth mentioning here since it supersedes the other add-ons, but happy to move it later next to Requester add-on.

Reply:
I’m fine with whichever as others see fit.